content/en/docs/contributing/google-season-of-docs/improve-navigation-and-structure/index.md

```diff
@@ -0,0 +1,22 @@
+# Proposal: Improve the navigation structure of the cert-manager website
+
+Right now the content is not designed with our target audiences in mind.
+For example a new user will not easily find a guide explaining how to install cert-manager on AWS and configure it for Let’s Encrypt.
+Nor will a Cluster Administrator easily find information about how to optimise cert-manager for a large cluster with many Certificates.
+The information exists but is spread across multiple pages and is often not at the obvious page.
+
+We would like a technical writer:
+1. to help us identify our target audiences and
+2. to identify the key tasks of each of these audiences and
+3. re-structure the cert-manager.io website with this in mind.
+
+For example, we have discussed the following audiences and tasks: Beginner, Cluster Administrator, User, Integrator, New Contributor
+and each of these people will be interested in a different set of tasks.
```

Comment:
Question: does defining the audience in the docs (as opposed to just having a list of named tasks) matter?

(Had this thought after looking at https://linkerd.io/2.11/tasks/ where it seems like task names are already descriptive enough to help the reader understand whether that's something they're interested in doing)

Reply:
I think the intent of this paragraph isn't to mandate the technical writer to define the audience in the docs, i.e., we don't need to have these audiences listed somewhere on the website. Instead, it is to study these audiences for the purpose of restructuring the website with these audiences in mind.